.github/workflows/update_assets.yaml

```diff
@@ -16,6 +16,8 @@ jobs:
         runs-on: ubuntu-latest
         env:
           GMAS_TEST_RUN: ${{ inputs.test || 'FALSE' }}
+          DSCI_AZ_SAS_DEV: ${{ secrets.DSCI_AZ_SAS_DEV }}
```

Comment:
Oh, was `DSCI_AZ_ENDPOINT` also not set here?

Reply:
Hah, it was also missing 😆 But I was testing and it still seems to cause some issues even when I put it back. So might just leave as is...